webextension/popup.js

```diff
@@ -101,6 +101,31 @@ function getShortCode(languageCode) {
     return languageCode.replace(/-.*/, "");
 }
 
+function suggestionClass(match) {
+    if(isSpellingError(match)){
```

Comment:
extra space needed after `if` before `(` to be consistent with the style of this file.
Likewise, after `)` and before `{`.